testsite/requirements.txt

```diff
@@ -1,15 +1,16 @@
-Django>=1.6
```

Comment:
Need to remove duplicate requirements otherwise ```pip install -r requirements.txt -r testsite/requirements.txt``` raise errors:
```Double requirement given: Django==1.7.6 (from -r testsite/requirements.txt (line 1)) (already in Django>=1.7 (from -r requirements.txt (line 1)), name='Django')```